bedevere/bpo.py

```diff
@@ -55,11 +51,27 @@
 async def new_label(gh, event):
     """Update the status if the "trivial" label was added."""
     if event.data["label"]["name"] == TRIVIAL_LABEL:
-        await _post_status(gh, event, TRIVIAL_STATUS)
+        issue_number_found = ISSUE_RE.search(
+            event.data["pull_request"]["title"])
+        if issue_number_found:
+            status = get_found_issue_number_status(issue_number_found)
+        else:
+            status = TRIVIAL_STATUS
+        await _post_status(gh, event, status)
 
 
 @router.route("pull_request", "unlabeled")
 async def removed_label(gh, event):
     """Re-check the status if the "trivial" label is removed."""
     if event.data["label"]["name"] == TRIVIAL_LABEL:
         await set_status(gh, event)
+
+
+def get_found_issue_number_status(found_issue):
```

Comment:
You're actually not "getting" a status but creating one. So maybe `create_success_status()` (technically ambiguous due to trivial issues, but they have their own static status so I think it's fine)?